docs: update Academy deploying section to use .actor directory structure (#2514)
apify/apify-docs: sources/academy/platform/deploying_your_code/index.md

## The "Actorization" workflow {#workflow}

Follow these main steps to turn a piece of code into an Actor:

1. Handle [accepting inputs and writing outputs](./inputs_outputs.md).

> 2. Create a [`.actor` directory](/platform/actors/development/actor-definition) with an `actor.json` file to define your Actor.

Reviewer: ```suggestion
2. Create the [`.actor` directory](/platform/actors/development/actor-definition) and an `actor.json` configuration file.
```
